README.md

```diff
@@ -48,38 +48,38 @@ The DuMuX-preCICE adapter should build fine if DuMuX, preCICE and their dependen
 
     - Note that extended features of DuMuX or the DuMuX-preCICE adapter may need additional DUNE modules.
 
-2. Download the DuMuX-preCICE adapter to the same directory as the DUNE modules and the `dumux` folder. At the moment there are no adapter releases, besides an outdated `v0.1` release, such the best way is to checkout the `develop` branch of the adapter.
+2. Download the DuMuX-preCICE adapter to the same directory as the DUNE modules and the `dumux` folder. It is recommended to use the latest release of the adapter, which can be found by checking out the relevant branch.
 
     ```text
-    git clone -b develop https://github.com/precice/dumux-adapter.git
+    git clone -b dumux-adapter-v1.0.0 https://github.com/precice/dumux-adapter.git
```

Comment:
Maybe it would be more natural to checkout the most current version via the tag since the branch `dumux-adapter-VERSIONNUMBER` might be deleted at some time. One could clone the repository like this instead
```suggestion
    git clone -b v1.0.0 https://github.com/precice/dumux-adapter.git
```
Then we would maybe need a hint to use the latest release based on the tag.

What is your opinion?

Reply:
So generally in the preCICE project repositories, we keep the release branches and don't delete them. But I agree that one could potentially delete it in the far future. Cloning the repository from a tag seems like a good idea. I will make sure that this tag in the documentation is updated at the time of a new release. Eventually I plan to write up a release guide where I will add this change as a step.